scripts/Tools/getTiming

```diff
@@ -86,7 +86,7 @@ class TimingParser:
         nprocs = 0
         ncount = 0
 
-        heading = heading_padded.strip()
+        heading = '"' + heading_padded.strip() + '"'
```

Comment:
The TimingParser class has been moved from scripts/Tools/getTiming to CIME.get_timing . So this change needs to be manually merged